docs/INSTALLATION.md

```diff
@@ -31,7 +31,7 @@ Open an administrative command prompt.  (If you need assistance opening an admin
 2.  Install the JDK:
 
      ```batchfile
-     C:\Windows\system32> choco install openjdk11
+     C:\Windows\system32> choco install openjdk
```

Comment:
Can we replace this with `choco install openjdk21`? We try to make sure that students are using a version of Java that is also supported by our Java test runner, so I'd prefer it if the instructions are written in such a way that students will install a specific version of Java, instead of the latest one.

Reply:
Unfortunately, there is no `openjdk21` Chocolatey package. I switched to the `temurin21` package.